compiler/ilgen/OMRIlBuilder.cpp

```diff
@@ -2604,7 +2604,29 @@ OMR::IlBuilder::Switch(const char *selectionVar,
                   uint32_t numCases,
                   JBCase **cases)
    {
-   TR::IlValue *selectorValue = Load(selectionVar);
```

Comment:
To be honest, I can no longer remember why we implemented `Switch` taking a condition "variable" and not just an `IlValue` :( . But I'm sure it's being used, so you're right that we shouldn't just remove it.